### ISSUE TITLE
QA: rename "vars" section in dvc.yaml (2.0) ?

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc#5164]
> Context: https://github.com/iterative/dvc.org/pull/2052#discussion_r548392124

"I like to call the values "variables" (though they are static 😕)" - @skshetry 

"Not sure about the naming - vars, etc...
`Parameters can also be used as variables` - this phrase sounds a bit confusing indeed"
\- @shcheklein 

In summary, it seems like a) we're going with the concept/term of "parameterization" and would prefer to avoid "variables" which is not really what these values are. They're parameters?

**Comment by jorgeorpinel** [comment]
What about calling it `params` for consistency and since they're not really "varaibles"?

```yaml
params:
  - script: hello.sh

stages:
  foo:
    cmd: source ${script}
```

**Comment by skshetry** [comment]
It's not clear what you are proposing.

**Comment by shcheklein** [comment]
@skshetry I guess, since you have all the information on this (discussions with Dmitry, Ruslan, research of similar tools), it would be helpful to share some context on how this name came about. What is your personal take, why do you like it - `vars`? May be share some examples from other tools from the top of your head. I can imagine, you have spent a few cycles to decide on this name, it doesn't come randomly.

E.g. a good answer -  we discussed `consts`, `params`, `vars`, etc and we realized that yes, vars is not ideal from the formal perspective (since they are not vars), but it is common in other templating languages + it is expected that it will be user friendly (and user case less about formalities).

**Comment by jorgeorpinel** [comment]
@skshetry I'm not proposing anything specific, which is why I labeled this a `discussion`.

As you know this comes from the docs PR (https://github.com/iterative/dvc.org/pull/2052#pullrequestreview-558334580) where I found that explaining the overlap between `params` and `vars` is difficult. That could signal to some confusion in the original intention/design. It's why sometimes we have to come back to the core repo during the doc writing phase sometimes.

We just have to think hard as a team here, I think... 🧠
